src/libotutil/ot-checksum-instream.c

```diff
@@ -1,3 +1,4 @@
+/* vi: set et sw=2 ts=2 cino=t0,(0: */
```

Comment:
Let's not do this, see instead https://github.com/ostreedev/ostree/pull/1206 - presumably there's a vim equivalent?  There's also `.editorconfig` which I think I used elsewhere.

Reply:
There is. I upstreamed my copy at https://github.com/ostreedev/ostree/pull/1208. @jlaako, feel free to suggest tweaks if there's things I'm missing.

Reply:
OK, I hope vim knows how to deal with it. I'll remove it.